### ISSUE TITLE
Add conversion seed data

### ISSUE THREAD

**Issue by blake-mealey** [blake-mealey/ConversionsServer#1]
Please add more unit types, units, and conversions to the DB seed data, and add a migration script for it.

See the setup steps in the project readme for getting started and don't hesitate to ask for help :)

- To add DB seed data, take a look at the ConversionsDb/SeedData files.
- To add a migration on Windows
  - In VS 2017, open Tools > NuGet Package Manager > Package Manager Console
  - Set ConversionsDb as startup project in Solution Explorer
  - Set ConversionsDb as default project in Package Manager Console
  - Run `Add-Migration -Name YourNameHere`
  - To test your migration, run `Update-Database`
- To add a migration on Ubuntu
  - run `dotnet ef migrations add YourNameHere`
  - To test your migration, run `dotnet ef database update`

**Comment by mykeels** [comment]
Is there a list of unit-types, units and conversions you're expecting for this issue?

**Comment by blake-mealey** [comment]
Not really. I originally just looked at another conversion app and copied a few of theirs over. Perhaps you could find another conversion app which has quote a few more units in it and add some from there. It would be best to focus on non SI units because I plan to handle those more dynamically in the future.
